README.md

```diff
@@ -1,10 +1,27 @@
-# Form Builder Publisher
 [![CircleCI](https://circleci.com/gh/ministryofjustice/fb-publisher/tree/master.svg?style=svg)](https://circleci.com/gh/ministryofjustice/fb-publisher/tree/master)
 
+# Form Builder Publisher
+
+This Ruby on Rails app allows users to deploy their forms in to [https://github.com/ministryofjustice/cloud-platform-environments](Cloud Platform)
+
+## Environments
+
+The app provides 2 environments to users. A `development` environment to test and preview changes and a `production` environment for public release
```

Comment:
Just a suggestion: A `development` (test) environment to preview changes and a `production` (live) environment for public release.